docs/userguides/deactivatedevices.md

```diff
@@ -0,0 +1,65 @@
+# Clean up your environment by deactivating devices
+
+Your Code42 environment may contain many old devices that are no longer active computers, and that have not connected to Code42 in quite some time. In order to clean up your environment, you can use the CLI to deactivate these devices in bulk.
```

Comment:
Lines are bit long - obviously only appears so in editors, but it doesn't hurt to put newlines in.

Reply:
I've added some line breaks. It's pretty arbitrary at this point so we may want to move them around later.